client/src/view/icons/Calendar.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+export default ({size = "30"}) => (
+<svg fill="#000000" height="30" viewBox="0 0 24 24" width="30">
```

Comment:
You are taking the size argument in fine however you aren't using it. 

```jsx
export default ({size = "30"}) => (
   <svg fill="#000000" height={size} viewBox="0 0 24 24" width={size}>
```

I personally like making the color configurable. So that would become
```jsx
export default ({ size = "30", color="#000" }) => (
   <svg fill={color} height={size} viewBox="0 0 24 24" width={size}>
``` 

After that, there is one last tiny gotcha
```jsx
export default ({ size = "30", color="#000", ...args }) => (
   <svg fill={color} height={size} viewBox="0 0 24 24" width={size} {...args}>
``` 
This takes any additional props on your component and adds them to the svg. If you don't do this you can't style your component with styled-components.